content/pt/docs/concepts/distributions.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Distribuições
+description: >-
+  Uma distribuição, que não deve ser confundida com um fork, é uma versão
+  customizada de um componente OpenTelemetry.
+weight: 190
+default_lang_commit: 2f34c456ab38b4d3502cd07bc36fa1455d4ef875
+---
+
+Os projetos OpenTelemetry consistem em múltiplos [componentes](../components)
```

Comment:
Pensando que aqui talvez faça sentido usar com gênero, algo como: "Os projetos **do** Opentelemetry", o que acha?

O mesmo vale pro campo `description` mais acima: "componente do OpenTelemetry".

Reply:
Faz sentido sim! Inclusive, me recordo de ter lido em algum lugar sobre utilizar gênero sempre que for mencionar o OpenTelemetry. Boa!